numpy/add_newdocs.py

```diff
@@ -21,7 +21,7 @@
 ###############################################################################
 
 add_newdoc('numpy.core', 'flatiter',
-    """
+           """
```

Comment:
You seem to have touched a lot of whitespace in this commit in a questionably useful way - can I request that you back these changes out of this PR?

Reply:
Oops, accidental autopep8 nonsense.